package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "auth-demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
You should include your debug functionality here so that you can use it in the terminal when you run npm test.